libs/blocks/section-metadata/section-metadata.css

```diff
@@ -137,6 +137,46 @@
   gap: var(--spacing-xxxl);
 }
 
+.section.border-radius-block-s > div,
+.section.border-radius-block-s > div .background picture img {
+  border-radius: var(--border-radius-s);
+}
+
+.section.border-radius-block-l > div,
+.section.border-radius-block-l > div .background picture img {
+  border-radius: var(--border-radius-l);
+}
+
+.section.border-radius-block-xl > div,
+.section.border-radius-block-xl > div .background picture img {
+  border-radius: var(--border-radius-xl);
+}
+
+.section.no-border-block-radius > div,
```

Comment:
Since this is supposed to negate the previous class names, shouldn't it be `no-border-radius-block`? I see the same pattern has been applied for the image use case.

Reply:
done. That was a mistake